packages/inter-protocol/src/auction/auctionMath.js

```diff
@@ -0,0 +1,67 @@
+import { AmountMath } from '@agoric/ertp';
+import {
+  ceilMultiplyBy,
+  floorDivideBy,
+} from '@agoric/zoe/src/contractSupport/index.js';
+
+/**
+ * @import {Amount} from '@agoric/ertp/src/types.js';
+ */
+
+/**
+ * @param {object} p
+ * @param {Amount<'nat'>} p.bidAlloc current allocation of the bidding set
+ * @param {Amount<'nat'>} p.collateralWanted want of the offer
+ * @param {Amount<'nat'>} p.collateralAvailable available to auction
+ * @param {Ratio} p.curAuctionPrice current auction price
+ * @param {Amount<'nat'> | null} p.remainingProceedsGoal amount still needing
+ *   liquidating over multiple rounds; null indicates no limit
+ */
+export const amountsToSettle = ({
+  bidAlloc,
+  collateralWanted,
+  collateralAvailable,
+  curAuctionPrice,
+  remainingProceedsGoal,
+}) => {
+  console.log('amountsToSettle', {
```

Comment:
unilaterally logging from library functions is something I usually avoid. Maybe  add a `log = console.log` property to `p`?